### PR TITLE
GL-366: Add admin APIs to list out a page of overrides

### PR DESCRIPTION
**Pull request by rasikasri**
### Jira link

[GL-366](https://transformuk.atlassian.net/browse/GL-366)

### What?

I have added/removed/altered:

- [ ] Added admin APIs to list / create / edit / remove overrides

### Why?

I am doing this because:

- So that a screen in the Admin area can be built to manage exempting certificate overrides

### CODE REVIEW

File: app/controllers/api/admin/green_lanes/category_assessments_controller.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ module Admin
     module GreenLanes
       class CategoryAssessmentsController < AdminController
         include Pageable
+        include XiOnly
```
Comment (jebw) [inline app/controllers/api/admin/green_lanes/category_assessments_controller.rb:6]: :raised_hands:

File: app/controllers/api/admin/green_lanes/exempting_certificate_overrides_controller.rb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            render json: serialize(eco),
+                   location: api_admin_green_lanes_exempting_certificate_override_url(eco.id),
+                   status: :created
+          else
+            render json: serialize_errors(eco),
+                   status: :unprocessable_entity
+          end
+        end
+
+        def update
```
Comment (jebw) [inline app/controllers/api/admin/green_lanes/exempting_certificate_overrides_controller.rb:32]: After refinement I don't think we need to support the `#update` so this action can be dropped

File: config/routes.rb [hunk 3]
```diff
@@ -68,6 +68,7 @@
         namespace :green_lanes do
           resources :category_assessments, only: %i[index show create update destroy]
           resources :themes, only: %i[index]
+          resources :exempting_certificate_overrides, only: %i[index show create update destroy]
```
Comment (jebw) [inline config/routes.rb:71]: `#update` can get dropped

**Review by jebw — CHANGES_REQUESTED** [review]
Looks good and great to see the XI checks split out

Only query is around the update which I think becomes irrelevant now we don't require a `behaviour` attribute

File: app/controllers/api/admin/green_lanes/exempting_certificate_overrides_controller.rb (hunk 2 above)
Comment (rasikasri) [reply app/controllers/api/admin/green_lanes/exempting_certificate_overrides_controller.rb]: Done

**Review by jebw — APPROVED** [review]
LGTM

**Pull request merged**